src/Microsoft.Performance.SDK/Processing/TableConfiguration.cs

```diff
@@ -262,7 +237,7 @@ public void AddColumnRole(ColumnRole columnRole, Guid columnGuid)
         /// <param name="columnRole">
         ///     The role to be removed.
         /// </param>
-        public void RemoveColumnRole(ColumnRole columnRole)
+        public void RemoveColumnRole(string columnRole)
```

Comment:
since we're here should we update this to return a `bool` and just return the .Remove below?

Reply:
We didn't do it before, not sure if it matters?

Reply:
just thought it'd be nice since we're currently just disregarding Remove's return value. Could be useful for error handling down the road.